managed/models/service_helpers_test.go

```diff
@@ -158,10 +158,11 @@ func TestServiceHelpers(t *testing.T) {
 		services, err = models.FindServices(q, models.ServiceFilters{NodeID: "N1"})
 		assert.NoError(t, err)
 		assert.Equal(t, 3, len(services))
-		assert.Equal(t, services, []*models.Service{{
+		assert.Equal(t, []*models.Service{{
```

Comment:
Note: the signature of `assert.Equal` is `(t, expected, actual, ...)` and not `(t, actual, expected)`. This puts it in the right order.

Reply:
I wonder if `assert.ElementsMatch` is a better option here (but fine if we can always guarantee that both services will be in this same order).

Reply:
We chose `assert.Equal` because we want to check whether services are appearing in the exact order they were added, and not in a random order.